test/passing/comments_in_record.ml.ref

```diff
@@ -39,8 +39,8 @@ type t =
         (* aaaaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbbbb
```

Comment:
I think the "weird formatting" comment is because the comment for the last field is not indented the same as for the other fields. You are making a regression here.
Also the other comments were not indented (https://github.com/mirage/irmin/pull/582/files#diff-4846077446d291e891a3940001669978R135)